src/routes/auth.route.ts

```diff
@@ -23,7 +23,11 @@ const authRoute = Router();
     updatePassword,
   );
   
-  authRoute.post('/auth/login', validateSchema(AuthSchema.login), Login);
+authRoute.post('/auth/login', validateSchema(AuthSchema.login), Login);
+  
+ authRoute.post('/auth/send-verification-email', sendEmailVerification);
```

Comment:
Can we add some validation on this route since we're getting things from the body?